### DISCUSSION TITLE
How to get generated SQL?

### DISCUSSION THREAD

**Discussion by vedantroy** [prisma/prisma#3982] [Q&A] [answered]
How do I get the SQL that a prisma query generates?

**Comment by wanjas** [comment, marked as answer]
I guess here is the answer you are looking for https://github.com/prisma/prisma/discussions/3967

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
